.gitignore

```diff
@@ -4,3 +4,4 @@ deploy/local-unprivileged.yaml
 deploy/local-openshift.yaml
 bin
 .idea
+.vscode
```

Comment:
https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer maybe?

Reply:
TIL this is a thing